<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-5KMFOFH9" failures="2" tests="2" name="Default test(failed)" time="22.765" errors="0" timestamp="2023-10-13T18:59:10 IST">
  <testcase classname="dataDrivenFramework.HrmDataDrivenTest" name="loginTest" time="10.303">
    <failure type="java.lang.AssertionError" message="Login fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login fail expected [true] but found [false]
at dataDrivenFramework.HrmDataDrivenTest.loginTest(HrmDataDrivenTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="dataDrivenFramework.HrmDataDrivenTest" name="loginTest" time="11.359">
    <failure type="java.lang.AssertionError" message="Login fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login fail expected [true] but found [false]
at dataDrivenFramework.HrmDataDrivenTest.loginTest(HrmDataDrivenTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Default test(failed) -->
